.github/dependabot.yml

```diff
@@ -3,7 +3,10 @@
 
 version: 2
 updates:
-  # Dependencies listed in .github/workflows/*.yml
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

Comment:
We do not want to update Go dependencies to their latest versions, since this package itself is a library (not the end product) and updating the dependencies unnecessarily may cause issues with software which uses opencontainers/selinux.

IOW in here we want to use older packages, and let the software that uses opencontainers/selinux to decide which versions of dependencies to use, rather than inflict the latest versions.